packages/storybook/stories/components/text/4_demo.stories.ts

```diff
@@ -0,0 +1,34 @@
+import { defineCustomElement } from '@ovhcloud/ods-components/dist/components/ods-text';
+import { ODS_TEXT_PRESETS } from '@ovhcloud/ods-components';
+import { html } from 'lit-html';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+import type { Meta } from '@storybook/web-components';
+
+defineCustomElement();
+
+const meta: Meta = {
```

Comment:
export directly? or you want to explicitly set the type?

Reply:
It's just for the autocomplete & get the type for help